### ISSUE TITLE
Reconsider the ultraviolet and gamma ray strings in the light spectrum diagram

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/molecules-and-light#419]
There was a discussion over slack about the i18n and a11y of these strings. Because of the hyphen, these strings are english centric and are difficult to translate. If a hyphen cannot be used in a language, the value will become too long and also be bad for a11y because it will shrink down too small:

Here is one example:
<img width="1752" height="1182" alt="Image" src="https://github.com/user-attachments/assets/fb552162-d482-4cf7-aeb1-97350adb8c3f" />

Consider rotating the string to make more space for it.
